styles/main.css

```diff
@@ -0,0 +1,651 @@
+* {
+  box-sizing: border-box;
+}
+
+/* FONTS */
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-regular-web.woff") format("woff");
+  font-weight: normal;
+}
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-medium-web.woff") format("woff");
+  font-weight: 500;
+}
+
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-bold-web.woff") format("woff");
+  font-weight: bold;
+}
+
+@font-face {
+  font-family: "chirp";
+  src: url("../fonts/chirp-heavy-web.woff") format("woff");
+  font-weight: 800;
+}
+
+body,
+html {
+  margin: 0;
+  font-family: "chirp";
+  font-weight: normal;
+  overscroll-behavior-y: none;
+  --gray: #f6f9f9;
+  --blue: #1da1f2;
+  --border-color: rgba(128, 128, 128, 0.145);
+}
+
+/* --------------------------------*/
+.font-chirp {
+  font-family: "chirp";
+}
+.h-100 {
+  height: 100%;
+}
+
+.w-100 {
+  width: 100%;
+}
+.twitter-color {
+  color: var(--blue);
+  fill: var(--blue);
+}
+
+.gray {
+  color: gray;
+}
+.il-b {
+  display: inline-block;
+}
+.b-0 {
+  border: none;
+}
+.b-light {
+  border: 2px solid var(--border-color);
+}
+
+.b-thin {
+  border: 1px solid var(--border-color);
+}
+.b-b-thin {
+  border-bottom: 1px solid var(--border-color);
+}
+.b-sd-thin {
+  border-left: 1px solid var(--border-color);
+  border-right: 1px solid var(--border-color);
+}
+
+.b-eg-thin {
+  border-top: 1px solid var(--border-color);
+  border-bottom: 1px solid var(--border-color);
+}
+
+.b-white {
+  border-color: white;
+}
+
+.b-r-sm {
+  border-radius: 20%;
+}
+
+.b-r-md {
+  border-radius: 30px;
+}
+
+.b-r-sm {
+  border-radius: 15px;
+}
+.pd-0 {
+  padding: 0;
+}
+
+.pd-sm {
+  padding: 10px;
+}
+
+.pd-md {
+  padding: 15px;
+}
+
+.pd-lg {
+  padding: 20px;
+}
+
+.pd-sd-sm {
+  padding-left: 10px;
+  padding-right: 10px;
+}
+.pd-sd-md {
+  padding-left: 15px;
+  padding-right: 15px;
+}
+
+.pd-eg-md {
+  padding-top: 15px;
+  padding-bottom: 15px;
+}
+
+.pd-sd-lg {
+  padding-left: 20px;
+  padding-right: 20px;
+}
+.mg-sd-sm {
+  margin-left: 10px;
+  margin-right: 10px;
+}
+
+.mg-sm {
+  margin: 10px;
+}
+
+.mg-eg-md {
+  margin-top: 15px;
+  margin-bottom: 15px;
+}
+
+.mg-eg-sm {
+  margin-top: 10px;
+  margin-bottom: 10px;
+}
+.mg-eg-xd {
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+.mg-eg-xs {
+  margin-top: 5px;
+  margin-bottom: 5px;
+}
+
+.mg-t-sm {
+  margin-top: 10px;
+}
+.mg-t-md {
+  margin-top: 15px;
+}
+.mg-0 {
+  margin: 0;
+}
+
+a {
+  color: inherit;
+  text-decoration: none;
+}
+.z-1 {
+  z-index: 1;
+}
+.grid-center {
+  height: 100%;
+  display: grid;
+  align-items: center;
+}
+.text-medium {
+  font-weight: 500;
+}
+.text-regular {
+  font-weight: normal;
+}
+
+.text-bold {
+  font-weight: bold;
+}
+
+.font-s-md {
+  font-size: 15px;
+}
+.font-s-lg {
+  font-size: 20px;
+}
+
+.font-s-xs {
+  font-size: 8px;
+}
+.font-s-sm {
+  font-size: 10px;
+}
+.font-s-rg {
+  font-size: 13px;
+}
```

Comment:
nice grouping of CSS classes that have similar effects. perhaps some comments such as "font size attributes" above the code would help structure the CSS even further